src/components/FeedbackPage/FeedbackPage.scss

```diff
@@ -0,0 +1,31 @@
+@import '~style/variables';
+
+.feedbackWrapper {
+  @include container;
+
+  padding: 48px 0;
+  background: $color-overview--020;
+}
+
+.title {
+  margin-bottom: 40px;
+  font-size: 44px;
+  font-weight: 300;
+}
+
+.paragraph {
+  font-size: 22px;
```

Comment:
This should be `font-size: 300` (as one of the 'allowed styles' in`/style-guide` in our extremely elegant typography system)

Reply:
Hmm... `font-weight`, right?